### PR TITLE
Bump to ruby 2.7.x

### PR DESCRIPTION
**Pull request by jmarrec**

### CODE REVIEW

File: openstudio_measure_tester.gemspec [hunk 1]
```diff
@@ -27,7 +27,7 @@ Gem::Specification.new do |spec|
   spec.executables = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
-  spec.required_ruby_version = '~> 2.5.0'
+  spec.required_ruby_version = '~> 2.7.0'
 
   spec.add_dependency 'git', '~> 1.6.0'
```
Comment (jmarrec) [inline openstudio_measure_tester.gemspec:32]: 1.8.1 is out

Comment (jmarrec) [reply openstudio_measure_tester.gemspec:32]: Is this an actual dep or should it be development_dependency?

File: openstudio_measure_tester.gemspec [hunk 2]
```diff
@@ -37,6 +37,6 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'rubocop-checkstyle_formatter', '~> 0.4.0'
   spec.add_dependency 'simplecov', '~> 0.18.2'
```
Comment (jmarrec) [inline openstudio_measure_tester.gemspec:38]: latest avail: rubocop 1.9.1, simplecov 0.21.2

Comment (tijcolem) [reply openstudio_measure_tester.gemspec:38]: newer versions of rubocop have a native dep with jaro-winkler. Despite openstudio cli having jaro_winkler, it still caused problems and we had to roll back tor ubocop (0.54.0).

Comment (nllong) [reply openstudio_measure_tester.gemspec:38]: yep, I think we are still stuck on 0.54.0

**Review by nllong — APPROVED** [review]
Changes look good, need unit tests to pass to be sure though.

**Comment by tijcolem** [comment]
CI runs are now passing using a test container that has OpenStudio v3.1.1 w/ ruby 2.7.2
`nrel/openstudio:3.1.1-alpha-ruby272-dev`

I also created a new branch [0.2.X-LTS](https://github.com/NREL/OpenStudio-measure-tester-gem/tree/0.2.X-LTS) in the event we need any patched releases that require ruby 2.5.5

**Pull request merged**